x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/multi_signal_inventory/index.tsx

```diff
@@ -148,8 +148,8 @@ function useServicesEntitiesDetailedStatisticsFetcher({
 export function MultiSignalInventory() {
   const [searchQuery, setSearchQuery] = React.useState('');
   const { services } = useKibana<ApmPluginStartDeps & ApmServices>();
-  const { mainStatisticsData, mainStatisticsStatus } = useServicesEntitiesMainStatisticsFetcher();
   const mainStatisticsFetch = useServicesEntitiesMainStatisticsFetcher();
+  const { mainStatisticsData, mainStatisticsStatus } = mainStatisticsFetch;
```

Comment:
hmmm that looks odd, why has it fixed the problem?

Reply:
Just a suggestion: How about this (just out of curiosity because at first sight I was confused by this change)

```ts
const { mainStatisticsData, mainStatisticsStatus } = useServicesEntitiesMainStatisticsFetcher();
```

and then here, instead of:

```ts
 const { timeseriesDataFetch } = useServicesEntitiesDetailedStatisticsFetcher({
    mainStatisticsFetch,
    services: mainStatisticsData.services,
  });
```
```ts
 const { timeseriesDataFetch } = useServicesEntitiesDetailedStatisticsFetcher({
    mainStatisticsData,
    mainStatisticsStatus,
    services: mainStatisticsData.services,
  });
```

Reply:
>we're calling useServicesEntitiesMainStatisticsFetcher twice.

ah that's true... I didn't notice that.

Reply:
@crespocarlos good suggestion. Updated here https://github.com/elastic/kibana/pull/189631/commits/974afc1599564f2522753dbb6252d0bfdfcdb111